pyuntl/form_logic.py

```diff
@@ -141,15 +146,15 @@ def __init__(self, **kwargs):
             'timePeriod': self.set_coverage_timePeriod,
         }
         # Create the adjustable form property
-        # (datastructure for adjusting form with javascript)
+        # (data structure for adjusting form with JavaScript).
         self.adjustable_form = self.get_adjustable_form(coverage_dispatch)
 
     def set_coverage_placeName(self):
-        """ Determine the properties for the placeName coverage field """
-        if self.solr_response and self.solr_response != 'error' and \
-                self.solr_response.response != 'error':
-            location_list = self.solr_response.get_location_list_facet()\
-                .facet_list
+        """Determine the properties for the placeName coverage field."""
+        if (self.solr_response and
+                self.solr_response != 'error' and
```

Comment:
The issue with multi-clause `if` statements in parentheses is that if you indent the lines after the first to align with the first clause, it causes a four space indent, and then that would be the same alignment as the block inside the if statement which causes visual indent issues.